tools/acrn-crashlog/README.rst

```diff
@@ -61,6 +61,96 @@ To install the build:
    $ cd acrn-crashlog
    $ sudo make install
 
+Enabling/Disabling
+******************
+
+To enable this tool:
+
+.. code-block:: none
+
+   $ sudo crashlogctl enable
+
+Then it will show:
+
+.. code-block:: none
```

Comment:
Shouldn't we use ``.. code-block:: console`` here @dbkinder ?

Reply:
You **could** indeed use ``console`` here.  That will format the output with white on black to make it look more like the output on a screen. So, yes.